packages/blob-storage/cmd/blob_catcher/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"math/big"
+	"strconv"
+
+	"github.com/taikoxyz/taiko-mono/packages/blob-storage/internal/indexer"
+	"github.com/taikoxyz/taiko-mono/packages/blob-storage/internal/logic"
+)
+
+func main() {
+	// Define command-line flags
+	pastEvents := flag.Bool("past_events", false, "Enable indexing past events")
```

Comment:
Recommended to use `github.com/urfave/cli/v2` to be consistent with other binaries.